rotkehlchen/tests/unit/test_inquirer.py

```diff
@@ -521,7 +521,10 @@ def test_punishing_of_oracles_works(inquirer):
                 assert coingecko_mock.called is True
                 assert defillama_mock.called is True
 
+        # move the current time forward and check that coingecko is still penalized
+        with freeze_time(datetime.utcfromtimestamp(ts_now() + ORACLE_PENALTY_TS // 2)):
+            assert inquirer._coingecko.is_penalized() is True
+
         # move the current time forward and check that coingecko is no longer penalized
-        with freeze_time(datetime.fromtimestamp(ts_now() + ORACLE_PENALTY_TS + 1)):
+        with freeze_time(datetime.utcfromtimestamp(ts_now() + ORACLE_PENALTY_TS + 1)):
```

Comment:
> because the first instance freezes to local time, which introduces additional delay (as seen from UTC) in ts_now()

Yeah, you're right. This is due to `freezegun` returning the frozen time irrespective of the datetime function called.